yt_dlp/postprocessor/ffmpeg.py

```diff
@@ -315,26 +316,38 @@ def make_args(file, args, name, number):
             args += self._configuration_args(self.basename, keys)
             if name == 'i':
                 args.append('-i')
-            return (
-                [encodeArgument(arg) for arg in args]
-                + [encodeFilename(self._ffmpeg_filename_argument(file), True)])
+            return [encodeArgument(arg) for arg in args] + [encodeFilename(self._ffmpeg_filename_argument(file), True)]
 
         for arg_type, path_opts in (('i', input_path_opts), ('o', output_path_opts)):
-            cmd += itertools.chain.from_iterable(
-                make_args(path, list(opts), arg_type, i + 1)
-                for i, (path, opts) in enumerate(path_opts) if path)
+            for i, (path, opts) in enumerate(path_opts):
+                if path:
+                    cmd += make_args(path, list(opts), arg_type, i + 1)
 
+        # Needed to track ffmpeg progress
+        cmd.extend(['-progress', 'pipe:1', '-stats_period', '0.1'])
+
+        if type(input_path_opts) == list:
+            file_path, _ = input_path_opts[0]
+        elif type(input_path_opts) == tuple:
+            file_path, _ = input_path_opts
+        else:
+            raise ValueError(f"input_path_opts should either be a tuple or a list but it is a '{type(input_path_opts)}'")
```

Comment:
this is wrong. If you are looking for the first input file, it should be:
```suggestion
        file_path = input_path_opts[0][0]
```
But I don't understand what you are trying to do here. `info_dict` passed to the hook should be the actual info_dict, not this dummy dict

Reply:
info_dict doesn't exist in this context and it doesn't exists everywhere where this function is called